README.md

```diff
@@ -423,6 +442,8 @@ Discuss in [#13](https://github.com/jackbsteinberg/std-toast/issues/13) and [#39
 
 ### Appearance customization
 
+TODO: link to demos.
```

Comment:
We have the demos, should we link to them here (along with a warning that they're still being worked on / only work in Chrome Canary)?

Reply:
Yeah, although as a separate PR.